ste/sender-blockBlob.go

```diff
@@ -104,6 +109,25 @@ func getVerifiedChunkParams(transferInfo *TransferInfo, memLimit int64, strictMe
 		return
 	}
 
+	if putBlobSize >= memLimit {
+		err = fmt.Errorf("Cannot use a put blob size of %.2fGiB. AzCopy is limited to use only %.2fGiB of memory",
+			toGiB(putBlobSize), toGiB(memLimit))
+		return
+	}
+
+	if putBlobSize >= strictMemLimit {
+		err = fmt.Errorf("Cannot use a put blob size of %.2fGiB. AzCopy is limited to use only %.2fGiB of memory, and only %.2fGiB of these are available for chunks.",
+			toGiB(putBlobSize), toGiB(memLimit), toGiB(strictMemLimit))
+		return
+	}
+
+	if putBlobSize > common.MaxPutBlobSize {
```

Comment:
We have no error check in the commands, but we have an override in the jptm, but we also have an error, that we probably will never hit, because of the override? If this is intended to be a sanity check, let's make it clear this is a sanity check.